bindata/network/openshift-sdn/monitor.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: sdn
+  name: monitor-sdn
+  namespace: openshift-sdn
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
Given that the metrics aren't behind TLS, we don't need the `bearerTokenFile` or the `tlsConfig`